config/locales/en.yml

```diff
@@ -25,6 +25,8 @@ en:
       published: "Published"
       last_updated: "Last updated"
       see_all_updates: "see all updates"
+      toggle_less: "Show fewer"
+      toggle_more: "more"
```

Comment:
There's some Rails magic you can do to include the number here as well. Eg I'm EN it's 11 more, but in other languages the number may be at the end or in the middle.